### PR TITLE
Suggest for AF Sidebar

### PR DESCRIPTION
**Pull request by Raemon**

### CODE REVIEW

File: packages/lesswrong/components/sunshineDashboard/SuggestAlignmentItem.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          </Components.SidebarInfo>
+          {post.postedAt && <Components.SidebarInfo>
+            {moment(new Date(post.postedAt)).fromNow()}
+          </Components.SidebarInfo>}
+        </div>
+        <Components.SidebarInfo>
+          Endorsed by { post.suggestForAlignmentUsers && post.suggestForAlignmentUsers.map(user=>user.displayName).join(", ") }
+        </Components.SidebarInfo>
+        { hover && <Components.SidebarActionMenu>
+          { !post.suggestForAlignmentUserIds || !post.suggestForAlignmentUserIds.includes(currentUser._id) ?
```
Comment (Discordius) [inline packages/lesswrong/components/sunshineDashboard/SuggestAlignmentItem.jsx]: Replace this with named variable, probably something like `userHasVoted`. Makes reading this a lot easier.

File: packages/lesswrong/components/sunshineDashboard/SuggestAlignmentList.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            <Components.OmegaIcon className={classes.icon}/> Suggestions for Alignment
+          </Components.SunshineListTitle>
+          {this.props.results.map(post =>
+            <div key={post._id} >
+              <Components.SuggestAlignmentItem post={post}/>
+            </div>
+          )}
+        </div>
+      )
+    } else {
```
Comment (Discordius) [inline packages/lesswrong/components/sunshineDashboard/SuggestAlignmentList.jsx]: I don't think this else statement actually does anything. Javascript without returns return null if I remember correctly.

Comment (Raemon) [reply packages/lesswrong/components/sunshineDashboard/SuggestAlignmentList.jsx]: Components need to return null or they fuck up if I recall correctly.

Comment (Discordius) [reply packages/lesswrong/components/sunshineDashboard/SuggestAlignmentList.jsx]: Yep, you're correct. My bad.

File: packages/lesswrong/components/sunshineDashboard/SuggestAlignmentItem.jsx (hunk 1 above)
Comment (Raemon) [reply packages/lesswrong/components/sunshineDashboard/SuggestAlignmentItem.jsx]: Done

**Pull request merged**